src/html/index.html

```diff
@@ -56,16 +58,54 @@ <h4>
     </h4>
 
     <div class="text">
-      O evento é 100% grátis! Basta aparecer lá e participar.
+      Nossos eventos são 100% grátis! 
+    </div>
+
+    <div class="text">
+      Se inscreva para o workshop do dia 08/dezembro! 
     </div>
 
     <div>
-      <a href="https://www.facebook.com/events/418622648327200/" class="button">evento no facebook</a>
+      <a href="https://goo.gl/forms/BCsYRBmSKm6brTi63" class="button" target="_blank">Inscrever-se</a>
     </div>
   </div>
 </section>
 
-<section class="speakers" id="speakers">
+<section class="workshop" id="workshop">
```

Comment:
Que legal que está revivendo o site ✨Eu acho que eventualmente seria legal poder fazer isso menos mecanicamente.

Reply:
Sim! A nossa ideia é fazer um site principal com todos os eventos que já passaram também e arrumar pra fazer menos mecanicamente o/

Reply:
Muito bom 🚀Nunca saiu da minha lista de to-dos, muito legal a iniciativa!